data/adapters/file_fileset_adapter.py

```diff
@@ -311,43 +356,65 @@ def query_fileset_files_props_encode(self, accession):
 
         preferred_assay_titles = set()
         assay_term_ids = set()
-        prediction = False
-        prediction_method = None
+        method = None
+        class_type = None
         if dataset_type == 'Annotation':
-            prediction, prediction_method = self.parse_annotation(
-                dataset_object, portal_url, prediction, prediction_method, software, preferred_assay_titles, assay_term_ids)
+            class_type, method = self.parse_annotation(
+                dataset_object, portal_url, class_type, method, software, preferred_assay_titles, assay_term_ids)
+            software_titles = ', '.join([software for software in software])
+            method = f'{method} using {software_titles}'
+        else:
+            class_type = 'experimental'
```

Comment:
really minor point: change to experiment as noun, to be more consistent with other two classes?

Reply:
Ok.